src/bootstrap/bin/rustdoc.rs

```diff
@@ -55,13 +56,9 @@ fn main() {
         arg.push(&linker);
         cmd.arg(arg);
     }
-    if env::var_os("RUSTDOC_FUSE_LD_LLD").is_some() {
+    if let Some(no_threads) = env::var_os("RUSTDOC_LLD_NO_THREADS") {
```

Comment:
```suggestion
    if let Some(no_threads) = env::var("RUSTDOC_LLD_NO_THREADS") {
```
If it's defined then it's UTF-8.